mypyc/test-data/irbuild-int.test

```diff
@@ -80,3 +80,49 @@ L14:
 L15:
 L16:
     return 12
+
+[case testMinIR]
+def f(x: int, y: int) -> int:
+    return min(x, y)
+[out]
+def f(x, y):
+    x, y :: int                              
+    r0 :: native_int                              
+    r1 :: bit                                
+    r2 :: native_int                              
+    r3, r4, r5 :: bit                        
+    r6 :: bool                               
+    r7 :: bit                                
+    r8 :: object                             
+    r9 :: str                                
+    r10, r11, r12, r13 :: object             
+    r14 :: int                               
+L0:                                          
+    r0 = x & 1                               
+    r1 = r0 == 0                             
+    r2 = y & 1                               
+    r3 = r2 == 0                             
+    r4 = r1 & r3                             
+    if r4 goto L1 else goto L2 :: bool       
+L1:                                          
+    r5 = x < y :: signed                     
+    r6 = r5                                  
+    goto L3                                  
+L2:                                          
+    r7 = CPyTagged_IsLt_(x, y)               
+    r6 = r7                                  
+L3:                                          
+    if r6 goto L4 else goto L5 :: bool       
+L4:                                          
+    return x                                 
+L5:                                          
+    return y                                 
+L6:                                          
+    r8 = builtins :: module                  
+    r9 = 'min'                               
+    r10 = CPyObject_GetAttr(r8, r9)          
+    r11 = box(int, x)                        
+    r12 = box(int, y)                        
+    r13 = PyObject_CallFunctionObjArgs(r10, r11, r12, 0)
+    r14 = unbox(int, r13)                    
+    return r14                               
```

Comment:
This is obviously wrong. However, the same code when compiled using mypyc separately gives a different IR. Here's what's generated:
```
def f_min(x, y): 
    x, y :: int 
    r0 :: int64
    r1 :: bit 
    r2 :: int64
    r3, r4, r5 :: bit 
    r6 :: bool
    r7 :: bit 
L0:
    r0 = x & 1 
    r1 = r0 == 0
    r2 = y & 1 
    r3 = r2 == 0
    r4 = r1 & r3
    if r4 goto L1 else goto L2 :: bool
L1:
    r5 = x < y :: signed
    r6 = r5
    goto L3
L2:
    r7 = CPyTagged_IsLt_(x, y)
    r6 = r7
L3:
    if r6 goto L4 else goto L5 :: bool
L4:
    inc_ref x :: int 
    return x
L5:
    inc_ref y :: int 
    return y
```
Not sure  why L6 gets generated in the test suite though :confused: